kubetest/gke.go

```diff
@@ -531,14 +550,19 @@ export KUBE_NODE_OS_DISTRIBUTION='%[3]s'
 		dumpValues[jsonKey] = string(out)
 	}
 	if len(errorMessages) > 0 {
-		return fmt.Errorf("errors while dumping ConfigMaps: %s", strings.Join(errorMessages, ", "))
+		errs = append(errs, fmt.Errorf("errors while dumping ConfigMaps: %s", strings.Join(errorMessages, ", ")))
+		return wrapErrors("DumpClusterLogs", errs...)
 	}
+
 	jsonDump, err := json.Marshal(dumpValues)
 	if err != nil {
-		return err
+		errs = append(errs, err)
+		return wrapErrors("DumpClusterLogs", errs...)
 	}
+
 	if err := ioutil.WriteFile(filepath.Join(localPath, "gke-configmap.json"), jsonDump, 0644); err != nil {
-		return err
+		errs = append(errs, err)
+		return wrapErrors("DumpClusterLogs", errs...)
 	}
 	return nil
```

Comment:
We need to check if `len(errs) == 0` here and return appropriate error based on that instead of returning `nil` here.

Reply:
Done.